src/util/crypto.ts

```diff
@@ -23,34 +37,34 @@ const HexEncodeArray = [
   "f"
 ];
 
-function strToUint8Array(str) {
-  return new TextEncoder("utf-8").encode(str);
+function strToUint8Array(str: string): Uint8Array {
+  return new TextEncoder().encode(str);
```

Comment:
It's correct that we removed `"utf-8"` here?

Reply:
Yes, I think so. `TextEncoder` constructor doesn't take any arguments and always uses `utf-8`
https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder